### PR TITLE
projects: ad9361: linux support

### PR DESCRIPTION
**Pull request by ranechita**
Added support for linux sockets for ad9361. Also added defines in iio_app_run so that it can be used when there is no uart.

### CODE REVIEW

File: iio/iio_app/iio_app.c [hunk 1]
```diff
@@ -195,7 +195,7 @@ int32_t iio_app_run(struct iio_app_device *devices, int32_t len)
 	if (status < 0)
 		return status;
 #endif
-
+#ifndef LINUX_PLATFORM
```
Comment (buha) [inline iio/iio_app/iio_app.c:198]: this commit looks good but please rephrase that commit title to something more meaningful

**Review by mchindri — COMMENTED** [review]
Some changes are already in https://github.com/analogdevicesinc/no-OS/pull/1027.
There is just one thing to fix in that PR and it can be merge, then rebase this branch.

**Comment by ranechita** [comment]
v2

- removed flags variable from socket_accept function

**Comment by ranechita** [comment]
v3

- rebased and solved merge conflicts

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

**Review by mchindri — APPROVED** [review]

**Pull request merged**